### PR TITLE
Versioning cleanup

### PR DESCRIPTION
**Pull request by Neviothr**
**When merged this pull request will:**
- do some clean up in versioning component.

### CODE REVIEW

File: addons/versioning/XEH_postInitServer.sqf [hunk 1]
```diff
@@ -18,23 +18,35 @@ FUNC(paranoid) = {
...
     _cfg = (configFile >> "CfgSettings" >> "CBA" >> "Versioning" >> _key);
-    _addon = if (isClass _cfg) then { if (isText (_cfg >> "main_addon")) then { getText (_cfg >> "main_addon") } else { _key + "_main" }; } else { _key + "_main" };
+    _addon = if (isClass _cfg) then {
+        if (isText (_cfg >> "main_addon")) then {
+            getText (_cfg >> "main_addon");
+        } else {
+            _key + "_main";
+        };
+    } else {
+        _key + "_main";
```
Comment (commy2) [inline addons/versioning/XEH_postInitServer.sqf]: Those three above are return values and to indicate that the semi-colon is usually left out.

File: addons/versioning/XEH_preInit.sqf [hunk 2]
```diff
@@ -3,37 +3,42 @@
...
+for "_i" from 0 to (count (CFGSETTINGS) - 1) do {
     _prefix = (CFGSETTINGS) select _i;
-    if (isClass _prefix) then
-    {
-        _cfgPatches = if (isText(_prefix >> "main_addon")) then { getText(_prefix >> "main_addon") } else { format["%1_main", configName _prefix] };
+    if (isClass _prefix) then {
+        _cfgPatches = if (isText (_prefix >> "main_addon")) then {
+            getText(_prefix >> "main_addon");
+        } else {
+            format ["%1_main", configName _prefix];
```
Comment (commy2) [inline addons/versioning/XEH_preInit.sqf]: Same. Also, space after getText.

File: addons/versioning/XEH_postInitClient.sqf [hunk 3]
```diff
@@ -5,7 +5,9 @@ SLX_XEH_STR spawn {
     if (!SLX_XEH_DisableLogging) then {
         private "_logMsg";
         _logMsg = "CBA_VERSIONING_SERVER: ";
-        [GVAR(versions_serv), { _logMsg = (_logMsg + format["%1=%2, ", _key, (_value select 0) joinString "."])}] call CBA_fnc_hashEachPair;
+        [GVAR(versions_serv), {
+            _logMsg = (_logMsg + format["%1=%2, ", _key, (_value select 0) joinString "."])}
```
Comment (commy2) [inline addons/versioning/XEH_postInitClient.sqf]: Missing ; EOL

Comment (commy2) [reply addons/versioning/XEH_postInitClient.sqf]: And the closing curly belong at the start of the next line (after indentation).

File: addons/versioning/XEH_postInitClient.sqf [hunk 4]
```diff
@@ -5,7 +5,9 @@ SLX_XEH_STR spawn {
     if (!SLX_XEH_DisableLogging) then {
         private "_logMsg";
         _logMsg = "CBA_VERSIONING_SERVER: ";
```
Comment (commy2) [inline addons/versioning/XEH_postInitClient.sqf]: Might as well use private keyword here now.

File: addons/versioning/XEH_postInit.sqf [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-*/
+// Basic, Generic Version Checking System - By Sickboy <sb_at_dev-heaven.net>
 
 if (!SLX_XEH_DisableLogging) then {
     private ["_logMsg", "_filter"];
     _logMsg = "CBA_VERSIONING: ";
     _filter = {if (_x isEqualType 1) then {[_x] call CBA_fnc_formatNumber} else {_x}};
-    [GVAR(versions), { _logMsg = (_logMsg + format["%1=%2, ", _key, ([_value select 0, _filter] call CBA_fnc_filter) joinString "."])}] call CBA_fnc_hashEachPair;
+    [GVAR(versions), {
+        _logMsg = (_logMsg + format ["%1=%2, ", _key, ([_value select 0, _filter] call CBA_fnc_filter) joinString "."])}
```
Comment (commy2) [inline addons/versioning/XEH_postInit.sqf]: Same

File: addons/versioning/XEH_postInit.sqf [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
 
-/*
-    Basic, Generic Version Checking System - By Sickboy <sb_at_dev-heaven.net>
-*/
+// Basic, Generic Version Checking System - By Sickboy <sb_at_dev-heaven.net>
 
 if (!SLX_XEH_DisableLogging) then {
     private ["_logMsg", "_filter"];
     _logMsg = "CBA_VERSIONING: ";
     _filter = {if (_x isEqualType 1) then {[_x] call CBA_fnc_formatNumber} else {_x}};
```
Comment (commy2) [inline addons/versioning/XEH_postInit.sqf]: private keyword

**Comment by commy2** [comment]
Stupid wall of pink.

**Comment by Dystopian** [comment]
why change newline to crlf?

**Pull request closed without merging**